### DISCUSSION TITLE
Explorer crashes with Windows 11 10.0.22621.2361. Was working fine before the cumulative update.

### DISCUSSION THREAD

**Discussion by lmqferreira** [valinet/ExplorerPatcher#2121] [General]
Faulting application name: explorer.exe, version: 10.0.22621.2361, time stamp: 0x480de713
Faulting module name: dxgi.dll, version: 22621.1992.56.3, time stamp: 0x64e6228c
Exception code: 0xc0000005
Fault offset: 0x000000000003ad19
Faulting process id: 0x0x52E0
Faulting application start time: 0x0x1D9ED5B8FA9D206
Faulting application path: C:\Windows\explorer.exe
Faulting module path: C:\Windows\dxgi.dll
Report Id: 81d2027e-97e5-44f2-a686-b5c2e6f8790a
Faulting package full name: 
Faulting package-relative application ID:

**Comment by tallman69** [comment]
10.0.22621.2361 is an insider build.  EP does not support insider builds.  If you want EP to work you need to uninstall that update (KB5030310).

**Comment by pyrates999** [comment]
In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.
